.github/workflows/deploy.yml

```diff
@@ -30,13 +30,18 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 16
-      - name: Install Yalc
-        run: sudo npm i yalc -g
-      - name: NPM Install
-        run: npm ci
+      - name: Checkout amazon-chime-sdk repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-samples/amazon-chime-sdk'
+          path: amazon-chime-sdk
+      - name: Install latest version of React library and JS SDK in meeting demo
+        run: |
+          pwd
```

Comment:
I assume this will be removed before merge?

Reply:
If you mean `pwd`, yes, I will remove it